src/imitation/util/logger.py

```diff
@@ -14,7 +14,7 @@
 def _build_output_formats(
     folder: str,
     format_strs: Sequence[str],
-) -> Sequence[sb_logger.KVWriter]:
+) -> List[sb_logger.KVWriter]:
```

Comment:
We generally try to go for the more generic type to keep flexibility of changing return type in future, e.g. making it a tuple actually seems reasaonable. Do we use the fact it's a list not a sequence anywhere (e.g. do we rely on mutability, `MutableSequence`)?

Reply:
The output of `_build_output_formats` is always passed to a `stable_baselines3.common.logger.Logger`, which requires the type `List[KVWriter]` and not `Sequence[KVWriter]` (even though a sequence would be enough).
You are right though, just because we messed up the type annotations in SB3, we should not start messing them up in `imitation` as well.
I reverted the type annotation and added an explicit cast to `list` to please the type checker.